recipe/meta.yaml

```diff
@@ -18,20 +18,20 @@ build:
 
 requirements:
   host:
-    - python >=3.4
+    - python >=3.5
     - setuptools
     - pip
   run:
-    - flask >=0.12
-    - flask-assets
+    - flask >=1.0
+    - flask-assets >=2.0
     - flask-turbolinks
     - jsmin
     - libsass
     - natsort
-    - python >=3.4
```

Comment:
Why remove this?

Reply:
@isuruf I was trying to follow the instructions here: https://conda-forge.org/docs/maintainer/adding_pkgs.html#requirements

I don't see `python` listed under the `run` category. It lists `python` under `host` and it lists "most python libraries" under `run` but not `python` which confused me. I will add it back.